### ISSUE TITLE
[Deque Analysis] Hyperlinks on non-white backgrounds have insufficient color contrast

### ISSUE THREAD

**Issue by tdonohue** [DSpace/dspace-angular#1149]
## Deque Analysis Summary

Two examples of this issue exist, spread across 10 issues in Deque analysis

1. Element has insufficient color contrast of 4.28 (foreground color: `#207698` , background color: `#e8ebf3`). Contrast must be 4.5 or more
   * **Location: Search Results page, all hyperlinks in the "Filters" sidebar (e.g. "show all", "collapse", date range links)**
   * Example: https://demo7.dspace.org/search?spc.page=1&query=test&spc.sf=score&spc.sd=DESC
   * Deque issue tickets IDs: 468498, 468499, 468503, 468504, 468505, 468506
2. Element has insufficient color contrast of 4.31 (foreground color: `#207698` , background color: `#e9ecef`). Contrast must be 4.5 or more
   * **Locations: Hyperlinks which appear in breadcrumbs (on many pages)**
   * Example: https://demo7.dspace.org/collections/8ad84c46-bdf1-4558-9b90-5c43ff396980
   * Deque issue ticket IDs: 469217, 469218, 469219, 469220

## Recommended Fix

Likely, this is most easily fixed by making the teal color slightly darker (perhaps for all links).  The color `#1F7293` seems sufficient for both backgrounds for WCAG AA standards according to Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

(Other recommendations welcome)

## More Information / Tools

* As necessary, see mentioned Issue IDs in our assessment results (requires login): https://axeauditor.dequecloud.com/test-run/0856438a-a19a-11eb-bc31-b7d5be387c86/issues
* Consider using [Deque's free Chrome plugin](https://chrome.google.com/webstore/detail/axe-devtools-web-accessib/lhdoppojpmngadmnindnejefpokejbdd?hl=en-US) to check your work
* Deque's color contrast tool: https://dequeuniversity.com/rules/axe/4.1/color-contrast?application=axeAPI

**Comment by tdonohue** [comment]
@atarix83 : Assigning this to you for analysis.  Might be a quick fix (make the color darker)

**Comment by michdyk** [comment]
@tdonohue Could you assign me to this issue? My team and I have investigated the accessibility of the system recently, so I guess it will be great to have done this task until we are fresh after our initial analysis.

**Comment by michdyk** [comment]
@tdonohue please assign my colleague @Dawnkai to this task. He is going to provide the PR for this issue. Also when it will be accepted, I kindly ask to give credits for this bug fix contribution to PCG Academia. Thanks
